source/settings/core.rst

```diff
@@ -2301,6 +2301,10 @@ Variable Long name      Description
 -        passdb:<name>  Return passdb extra field "name". %{passdb:name:default} returns "default" if "name" doesn't exist (not returned if name exists but is empty) (v2.2.19+)
 ======== =============  =====================================================================================================
 
+Note: The ``client_id`` is only useful when proxying and needs to have the
+:ref:`setting-imap_id_retain` variable enabled. For logging this field on the
+client directly use :ref:`setting-imap_id_log`.
```

Comment:
The client_id can be useful for non-proxying purposes also. Also I started wondering about this setting in general and did somewhat bigger cleanups: https://github.com/dovecot/documentation/pull/363

So this text could just be removed here.

Reply:
Removed.